src/de/blau/android/propertyeditor/TagEditorFragment.java

```diff
@@ -708,7 +710,7 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
 					if (isChecked) {
 						tagSelected();
 					} else {
-						tagDeselected();
+						relationDeselected();
```

Comment:
No. I had to come up with a common interface as defined in `AccessibleRelation` implemented by `TagEditorFragment`, `RelationMembersFragment` and `RelationMembershipFragment`.